WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersPeriodViewController.swift

```diff
@@ -65,20 +65,35 @@ class StoreStatsAndTopPerformersPeriodViewController: UIViewController {
         return StoreStatsV4PeriodViewController(timeRange: timeRange, currentDate: currentDate)
     }()
 
+    private lazy var inAppFeedbackCardViewController = InAppFeedbackCardViewController()
+
     private lazy var topPerformersPeriodViewController: TopPerformerDataViewController = {
         return TopPerformerDataViewController(granularity: timeRange.topEarnerStatsGranularity)
     }()
 
     // MARK: Internal Properties
 
     private var childViewContrllers: [UIViewController] {
-        return [storeStatsPeriodViewController, topPerformersPeriodViewController]
+        return [storeStatsPeriodViewController, inAppFeedbackCardViewController, topPerformersPeriodViewController]
     }
 
-    init(timeRange: StatsTimeRangeV4, currentDate: Date) {
+    private let featureFlagService: FeatureFlagService
+
+    /// If applicable, present the in-app feedback. The in-app feedback may still not be presented
+    /// depending on the constraints. But setting this to `false`, will ensure that it will
+    /// never be presented.
```

Comment:
nit: maybe these comments could be in the public initializer so that it's visible to the caller? feel free to include just `canDisplayInAppFeedback`

Reply:
Thanks! Added in c006b7900164fc36a2d35cd1324cb6e78a5625aa.